x-pack/plugins/ml/server/routes/data_visualizer.ts

```diff
@@ -83,7 +82,7 @@ export function dataVisualizerRoutes({ router, getLicenseCheckResults }: RouteIn
       path: '/api/ml/data_visualizer/get_field_stats/{indexPatternTitle}',
       validate: dataVisualizerFieldStatsSchema,
     },
-    licensePreRoutingFactory(getLicenseCheckResults, async (context, request, response) => {
+    mlLicense.fullLicenseAPIGuard(async (context, request, response) => {
```

Comment:
Would basic license check be the correct one for data visualizer?

Reply:
yes! good spot.
This also means some other endpoints will need relaxing to basic checks